paddle/fluid/framework/ir/graph.cc

```diff
@@ -152,6 +152,39 @@ void Graph::ResolveHazard(
   }
 }
 
+std::shared_ptr<Graph> Graph::Clone() {
```

Comment:
no test?

Reply:
Have added the executor test for the graph clone API.